src/components/Cards/detailPageCard.js

```diff
@@ -0,0 +1,56 @@
+import { Modale } from "../Playlist Modal/playListModal";
+import { LikeButton } from "../Buttons/like";
+import './detailPageCard.css'
+
+export const DetailPageCard = ({ video }) => {
```

Comment:
filename should also reflect the same naming as the component name. also components can have .jsx extension to separate them from normal js file. read about it.